tests/packagedcode/test_pypi.py

```diff
@@ -221,9 +221,9 @@ def test_parse_with_unpacked_wheel_meta_v21_1(self):
         self.check_packages_data(package, expected_loc, regen=REGEN_TEST_FIXTURES)
 
     def test_parse_with_unpacked_wheel_meta_v21_2(self):
-        test_file = self.get_test_loc('pypi/unpacked_wheel/metadata-2.1/pip-20.2.2.dist-info/METADATA')
```

Comment:
You cannot remove this test and the one below. We still need to support plain unpacked wheels, even if not under a site-packages dir.